src/Axiom/Set.agda

```diff
@@ -167,9 +167,18 @@ record Theory {ℓ} : Type (sucˡ ℓ) where
   ∈-unions : {a : A} {U : Set (Set A)} → (∃[ T ] T ∈ U × a ∈ T) ⇔ a ∈ proj₁ (unions U)
   ∈-unions = proj₂ $ unions _
 
+  fromList-finite : ∀ {l : List A} → finite (fromList l)
```

Comment:
This is an easy proof (that a set created from a list is finite), but I found it helpful at one point.  After moving to a more list-based approach in this PR, it's no longer needed.  I wonder if I should delete it or leave it for future reference/use.